src/GovUk.Education.ExploreEducationStatistics.Admin/Services/Interfaces/Cache/ICacheKeyService.cs

```diff
@@ -0,0 +1,14 @@
+using System;
```

Comment:
I think we are generally adding `nullable enable` to all new classes and wherever else we can.

Reply:
Done, thanks!